packages/contracts-ops/contracts/Config.sol

```diff
@@ -0,0 +1,202 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import {UpgradeBeaconProxy} from "@nomad-xyz/contracts-core/contracts/upgrade/UpgradeBeaconProxy.sol";
+import {UpgradeBeacon} from "@nomad-xyz/contracts-core/contracts/upgrade/UpgradeBeacon.sol";
+import {UpgradeBeaconController} from "@nomad-xyz/contracts-core/contracts/upgrade/UpgradeBeaconController.sol";
+import {UpdaterManager} from "@nomad-xyz/contracts-core/contracts/UpdaterManager.sol";
+import {XAppConnectionManager} from "@nomad-xyz/contracts-core/contracts/XAppConnectionManager.sol";
+
+import "forge-std/Vm.sol";
+import "forge-std/Test.sol";
+
+abstract contract Config {
```

Comment:
qq: is the intention to use this in conjunction with the `NomadProtocol` contract structure? Asking because the intention behind that was to be able to load fork tests (via config) AND locally deployed integration tests within the same interface, so tests could be easily used for both (RFC: https://app.clickup.com/14205595/v/dc/dhgmv-26427/dhgmv-25167 )

Reply:
if i understand correctly, the entire purpose of this class is parsing values from the config in a clean way - it'll be used in the fork fixture to instantiate `NomadProtocol` from a config?

Reply:
(perhaps can add a little documentation to the top of the contract as well)

Reply:
gonna document all in the readme, with updater rotation as an example